test/e2e/apimachinery/webhook.go

```diff
@@ -256,6 +257,10 @@ func registerWebhook(f *framework.Framework, context *certContext) {
 	By("Registering the webhook via the AdmissionRegistration API")
 
 	namespace := f.Namespace.Name
+	// Create a webhook that always fails (fail open)
```

Comment:
We use "always fails" to describe both "fail open" and "fail closed".  (Other two marked below).  I don't actually understand what this test is trying to test.  Maybe we can add a short comment that explain the overall test plan and behavior that we're trying to verify?